packages/checkbox-group/src/LionCheckboxIndeterminate.js

```diff
@@ -0,0 +1,110 @@
+import { html, css } from '@lion/core';
+import { LionCheckbox } from './LionCheckbox.js';
+
+export class LionCheckboxIndeterminate extends LionCheckbox {
+  static get properties() {
+    return {
+      /**
+       * Indeterminate state of the checkbox
+       */
+      indeterminate: {
+        type: Boolean,
+        reflect: true,
+      },
+    };
+  }
+
+  get _checkboxGroupNode() {
+    return /** @type {import('./LionCheckboxGroup').LionCheckboxGroup} */ (this.parentElement);
+  }
+
+  get _subCheckboxes() {
```

Comment:
Hi @MatthieuLebigre,

Looking good so far!

In the end we need also support something like:

```html
<lion-checkbox-group name="scientists[]" label="Favorite scientists">
  <lion-checkbox-indeterminate label="Old Greek scientists">
    <lion-checkbox label="Archimedes" .choiceValue=${'Archimedes'}></lion-checkbox>
    <lion-checkbox label="Plato" .choiceValue=${'Plato'}></lion-checkbox>
    <lion-checkbox label="Pythagoras" .choiceValue=${'Pythagoras'}></lion-checkbox>
  </lion-checkbox-indeterminate>
  <lion-checkbox-indeterminate label="17th Century scientists">
    <lion-checkbox label="Isaac Newton" .choiceValue=${'Isaac Newton'}></lion-checkbox>
    <lion-checkbox label="Galileo Galilei" .choiceValue=${'Galileo Galilei'}></lion-checkbox>
  </lion-checkbox-indeterminate>
</lion-checkbox-group>
```

So `_subCheckboxes` should only get its own child `formElements`.

Reply:
yea so perhaps:
```js
return this._checkboxGroupNode.formElements.filter(checkbox => checkbox !== this && this.contains(checkbox));
```